packages/devtools_app/lib/src/inspector/flutter/summary_tree_debug_layout.dart

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+import '../../inspector/inspector_text_styles.dart' as inspector_text_styles;
+import '../diagnostics_node.dart';
+import 'inspector_data_models.dart';
+
+class ConstraintsDescription extends AnimatedWidget {
+  const ConstraintsDescription({
+    this.diagnostic,
+    AnimationController animationController,
+    Key key,
+  }) : super(key: key, listenable: animationController);
+
+  final RemoteDiagnosticsNode diagnostic;
+
+  String describeDimension(double min, double max, String dim) {
```

Comment:
rename dim  and Dimension to `axis` as I think that is more consistent with Flutter.
in general, avoid abbreviations.

Reply:
ok, to be honest I was copy pasting this piece of code from Flutter code :joy: